src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -30,15 +32,18 @@ const Sidebar = () => {
   const isEnabled = chatUsers && privateChats;
 
   const chatUserIds = privateChats
-    ?.flatMap((chat) => [chat.from, chat.to])
+    ?.sort(chatSort)
+    .slice(0, NUM_CHAT_UIDS_TO_LOAD)
```

Comment:
This should probably be pushed back to the firestore query we're using to load the private chats, or at the very least within the selector function (so it keeps memo unless that slice we are interested in changes), rather than just filtering them out here on the frontend.

Reply:
Yes it should. I’ll fix that.

Reply:
Yep this makes sense to push back into the selector, it’ll also make the result of the selector more stable.

Reply:
Agreed